src/game.cpp

```diff
@@ -139,7 +140,7 @@ void Game::checkForNewMill() {
 /*Updates list of mills and sets capture mode if new mill found*/
     unsigned int i;
     std::vector<int> newMills;
-    for (i = 0; i < millList.size(); i++) {
+    for (i = 0; i < millList.size(); i++) { // Iterate through sets of spaces that form mills and check if occupied by all white or black pieces
```

Comment:
looking through the PR so far, the comments have been on the line proceeding the code. I think it would be best to follow a standard format for code comments where we always have the comments on the line before the code or on the same line.